lib/transport.js

```diff
@@ -0,0 +1,29 @@
+'use strict'
+
+const { createRequire } = require('module')
+const caller = require('get-caller-file')
+const { isAbsolute } = require('path')
+
+const ThreadStream = require('thread-stream')
+
+function transport (filename, workerData) {
+  if (!(isAbsolute(filename) || filename.indexOf('file://') === 0)) {
+    const callerFile = caller()
+    const callerRequire = createRequire(callerFile)
+    filename = callerRequire.resolve(filename)
+  }
+
+  const stream = new ThreadStream({
+    filename,
+    workerData,
+    sync: true // TODO should this be configurable?
```

Comment:
should other worker thread options be configurable (e.g. supplying a --loader flag)

Reply:
now is fixed, check it out